prime-angular-frontend/src/app/modules/site-registration/shared/components/business-licence-expiry/business-licence-expiry.component.html

```diff
@@ -0,0 +1,30 @@
+<ng-container [formGroup]="form">
+  <app-page-subheader2>
+    <ng-container appPageSubheader2Title>Business Licence Expiry Date</ng-container>
+    <ng-container appPageSubheader2Summary>
+      Use the calendar option below to tell us when this business licence expires.
+    </ng-container>
+  </app-page-subheader2>
+
+  <div class="row">
+    <div class="col">
+
+      <mat-form-field class="w-100">
+
+
+
+        <input matInput
+               placeholder="Expiry Date"
+               [matDatepicker]="expiryDatePicker"
+               formControlName="businessLicenceExpiry"
+               (click)="expiryDatePicker.open()"
+               readonly>
+
+        <mat-datepicker #expiryDatePicker
+                        startView="multi-year"
+                        disabled="false"></mat-datepicker>
```

Comment:
seems should have a `[min]` restriction on the input so user can not input a expiry date in the past.